presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -269,6 +272,9 @@ protected void setup(Binder binder)
         if (serverConfig.isResourceManager()) {
             install(new ResourceManagerModule());
         }
+        else if (serverConfig.isCatalogServer()) {
+            install(new CatalogServerModule());
+        }
```

Comment:
same comment as above. What if we wanna colocate resource manager and catalog server in the same node?